web/src/services/api.ts

```diff
@@ -343,6 +334,27 @@ export abstract class Api {
             });
         };
     }
+    private static getClientIP(req: Request) {
+        var client_ip = (typeof req.headers['x-forwarded-for'] === 'string'
+                && req.headers['x-forwarded-for'].split(',').shift())
+                || req.connection?.remoteAddress;
+        if(client_ip && client_ip.includes("::ffff:")){
```

Comment:
client_ip.includes("::ffff:") This check is not required

Reply:
Done.